### PR TITLE
#124 optimiser le code avec des composants

### PR DESCRIPTION
**Pull request by dejuanxavier**

### CODE REVIEW

File: src/components/LoadingPopUp.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+         * activates the modal until the value is equal.
+         * @param {*} val
+         * @author Xavier de Juan
+         */
+        loadingPopUp(val) {
+            if (!val) return
+        },
+    },
+    methods: {
+        handleLoadPopUpShow() {
```
Comment (MarcoTribuz) [inline src/components/LoadingPopUp.vue]: Change name function to --> hidePopup or showPopup for example :)

File: src/router/index.js [hunk 2]
```diff
@@ -34,6 +35,11 @@ const routes = [
     name: '/xml-View',
     component: XMLParser
   },
+  {
```
Comment (MarcoTribuz) [inline src/router/index.js]: why? :)

File: src/libs/consts.js [hunk 3]
```diff
@@ -25,5 +36,9 @@ export {
     XML_PRTRY,
     XML_NTRY_REF,
     XML_CREDITOR_ACCOUNT,
-    XML_ULTIMATE_DEBTOR
+    XML_ULTIMATE_DEBTOR,
+    // SHOWLOADPOPUP,
```
Comment (MarcoTribuz) [inline src/libs/consts.js]: Delete unused comments :)

File: src/components/LoadingPopUp.vue [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+                <v-card-text>
+                    Veuillez patienter, en attente de réception
+                    <v-progress-linear class="mb-0" color="white" indeterminate></v-progress-linear>
+                </v-card-text>
+            </v-card>
+        </v-dialog>
+    </v-row>
+</template>
+<script>
+// import { SHOWLOADPOPUP } from '../libs/consts.js'
```
Comment (MarcoTribuz) [inline src/components/LoadingPopUp.vue]: Remove unused comments :)

**Review by MarcoTribuz — CHANGES_REQUESTED** [review]
Can you change something in your code as described below? Thaaaaank you :)

File: src/components/LoadingPopUp.vue (hunk 1 above)
Comment (dejuanxavier) [reply src/components/LoadingPopUp.vue]: Done

File: src/router/index.js (hunk 2 above)
Comment (dejuanxavier) [reply src/router/index.js]: Done

File: src/libs/consts.js (hunk 3 above)
Comment (dejuanxavier) [reply src/libs/consts.js]: Done

File: src/components/LoadingPopUp.vue (hunk 4 above)
Comment (dejuanxavier) [reply src/components/LoadingPopUp.vue]: Done

**Review by MarcoTribuz — APPROVED** [review]
Ok c'est tout bon

**Pull request merged**